build.sh

```diff
@@ -135,12 +135,28 @@ build_native_corert()
     echo "CoreRT native components successfully built."
 }
 
+get_current_linux_distro() {
+    # Detect Distro
+    if [ "$(cat /etc/*-release | grep -cim1 ubuntu)" -eq 1 ]; then
```

Comment:
Just FYI, you can also get the distro+version id using $(source /etc/os-release; echo $ID.$VERSION_ID). The os-release can be processesed as a script that sets various variables.